dist-build/android.sh

```diff
@@ -0,0 +1,44 @@
+#!/bin/sh
+
+# uncomment below to disable the minimal build
+#export LIBSODIUM_FULL_BUILD=TRUE
+# uncomment below and change the install location(s) to where you keep your ndk libraries and includes
```

Comment:
The comment is a bit misleading. I initially thought it represented the path where the NDK was installed.

Is the install prefix necessary? Android libraries will not run on the host operating system, so installing them in `/usr/local` sounds a bit strange. Is there anything wrong with installing the files locally, and then people will move them wherever they want?

Reply:
It does sound strange to me I misunderstood a previous comment of yours and thought this was the approach you preferred for cross compilation ie to match make install's layout.  The prefix can be set to anything when I use it I set it to /aSharedDrive/ndk-root/ I think it actually does work with /usr/local because all of the libs are inside folders that specify the architecture and operating system.

Reply:
this makes more sense in the context of my reply about the directory structure layout but how about I change the example prefix to "${ANDROID_NDK_HOME}/toolchains/llvm/prebuilt/$(uname | tr '[:upper:]' '[:lower:]')-x86_64/sysroot" and add another one to /custom/build/location ?